app/views/idv/session_errors/throttled.html.erb

```diff
@@ -27,5 +20,31 @@
               ),
             ) %>
       </p>
+      <p>
+        <strong>
+          <% if decorated_session.sp_name.present? %>
+            <%= link_to(
+                  t(
+                    'idv.failure.exit.with_sp',
+                    app_name: APP_NAME, sp_name: decorated_session.sp_name,
+                  ),
```

Comment:
Since the link is the same, is it possible to set a variable for the link text, and then just have one `link_to` statement?

Reply:
resolved as per pairing session  - unused keys issues arise in trying to merge into variable